src/components/promo/About.tsx

```diff
@@ -2,15 +2,60 @@ import Image from "next/image";
 
 function About() {
   return (
-    <div className="relative h-screen flex-col items-center justify-center overflow-hidden bg-hw-linear-gradient">
+    <div className="relative flex-col items-center justify-center overflow-hidden bg-hw-linear-gradient">
       {/* Grain Filter */}
-      <div className="absolute left-0 top-0 h-full w-full">
+      <div className="relative left-0 top-0 h-[20vh] w-full sm:h-[50vh]">
         <Image
           className="select-none opacity-20"
           src="/images/hwfilter.png"
           alt="Hack Western Main Page"
           fill
         />
+        <div className="absolute z-40 h-[50%] w-[100%] animate-cloud2 delay-75 md:h-[60%] lg:h-[75%] xl:h-[90%]">
+          <Image
+            src="/images/aboutcloud.svg"
+            alt="hack western cloud"
+            className="object-contain object-right"
+            fill
+          />
+        </div>
+        <div className="relative z-30 mx-auto flex h-full w-[90%] lg:w-[50%]">
+          <Image
+            src="/images/aboutticket.svg"
+            alt="hack western cloud"
+            className="object-contain"
```

Comment:
the shadow is weird...

<img width="862" alt="image" src="https://github.com/user-attachments/assets/3db392d2-5b99-4ea7-bb86-cf3d3e36ea6a">